doc/en/components/grids/_shared/row-adding.md

```diff
@@ -153,9 +154,11 @@ Then define a `{ComponentName}` with bound data source, `RowEditable` set to tru
     </igc-action-strip>
 </{ComponentSelector}>
 ```
+<!-- ComponentStart: TreeGrid -->
```

Comment:
Тhis should be closing tag 'component end' tag because above you have opening 'component start'.

`<!-- ComponentStart: TreeGrid -->`
```html
<{ComponentSelector} id="treeGrid" primary-key="ID" foreign-key="ParentID" row-editable="true">
    <igc-column field="Name" data-type="String"></igc-column>
    <igc-column field="Title" data-type="String"></igc-column>
    <igc-column field="HireDate" data-type="Date"></igc-column>
    <igc-column field="OnPTO" data-type="Boolean" width="130px">
    </igc-column>
    <igc-column field="Age" data-type="Number"></igc-column>
    <igc-action-strip id="actionstrip">
        <igc-grid-editing-actions add-row="true">
        </igc-grid-editing-actions>
    </igc-action-strip>
</{ComponentSelector}>
```
`<!-- ComponentStart: TreeGrid -->`